tests/TestRvsdgs.cpp

```diff
@@ -2192,9 +2192,8 @@ PhiTest2::SetupRvsdg()
   };
 
   auto SetupPhi = [&](
-    lambda::output & lambdaEight,
-    lambda::output & lambdaI)
-  {
+    lambda::output &lambdaEight,
+    lambda::output &lambdaI) {
```

Comment:
and here.

Reply:
Overhauled the TestRvsdgs.hpp and TestRvsdgs.cpp